src/Controls/src/Core/Compatibility/Handlers/TabbedPage/iOS/TabbedRenderer.cs

```diff
@@ -94,6 +94,7 @@ public void SetElement(VisualElement element)
 			UpdateBarTextColor();
 			UpdateSelectedTabColors();
 			UpdateBarTranslucent();
+			UpdateFlowDirection();
```

Comment:
The `UpdateFlowDirection()` call in SetElement is added after `UpdateBarTranslucent() `but before `UpdatePageSpecifics()`, requires this order? If is the case, we need to explore why, and if mandatory include some comments in the code.